cabal.project

```diff
@@ -48,6 +65,113 @@ constraints:
   , protolude < 0.3.1
   , algebraic-graphs < 0.7
 
+constraints:
```

Comment:
I would add a comment here explaining why we need those, and what needs to be done to be able to remove them in the future.

Reply:
Yes, I will do it. We did something similar for the other repositories.